awesome_cart/awesome_cart/doctype/awc_item_group/awc_item_group.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2018, DigitThinkIt, Inc. and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('AWC Item Group', {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['frappe' is not defined. (no-undef)](https://www.codacy.com/app/vjfalk/awesome_cart/pullRequest?prid=1269930)